rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf

```diff
@@ -1362,15 +1361,15 @@ SecRule &REQUEST_HEADERS:User-Agent "@eq 0" \
     tag:'capec/1000/210/272',\
     tag:'PCI/6.5.10',\
     tag:'paranoia-level/2',\
-    ver:'OWASP_CRS/3.3.5',\
+    ver:'OWASP_CRS/3.3.6',\
     severity:'NOTICE',\
     setvar:'tx.anomaly_score_pl2=+%{tx.notice_anomaly_score}'"
 
 
 #
 # PL2: This is a stricter sibling of 920120.
 #
-SecRule FILES_NAMES|FILES "@rx ['\";=]" \
+SecRule FILES_NAMES|FILES "@rx ['\";=\x5c]" \
     "id:920121,\
```

Comment:
The same for this rule: "updated patterns for rules X, Y, and Z to combat multipart bypass" or something?